xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -176,6 +178,7 @@ void CPVRClients::Unload(void)
   m_strPlayingClientName = "";
 
   m_clientMap.clear();
+  m_loadedEvent.Reset();
```

Comment:
hmm, why is it gone. now you would kill a slow but vital init process in case of timeout.